.github/workflows/velox.yml

```diff
@@ -0,0 +1,76 @@
+name: Velox Nightly Build and Test
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  container:
+    runs-on: ubuntu-latest
+    container: prestocpp/velox-velox-torcharrow:kpai-20220301
+    steps:
+      - name: Print CPU info
+        run: cat /proc/cpuinfo
+
+      - name: Setup Python environment
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Check out source repository
+        uses: actions/checkout@v2
+        with:
+            submodules: true
+
+      - name: Update submodules
+        run: |
+          git submodule update --init --recursive
+          cd csrc/velox/velox/
+          git submodule update --remote --merge
+
+      - name: Setup conda environment
+        run: | 
+          source /opt/conda/etc/profile.d/conda.sh
+          conda create --name ta_python37 python=3.7 -y
```

Comment:
@ejguan : Wondering if there is any matrix magic so we can build for Python 3.7/3.8/3.9? ^_^

Reply:
Yeah. As TA has pybind, you need to have a matrix of python versions to generate corresponding nightly wheel. Reference:
- Matrix definition: https://github.com/pytorch/data/blob/823d216535782a2c31a19bcfaeb450597a53f8c4/.github/workflows/_build_test_upload.yml#L62-L70
- Reference python version in steps: https://github.com/pytorch/data/blob/823d216535782a2c31a19bcfaeb450597a53f8c4/.github/workflows/_build_test_upload.yml#L72-L75
In this use case, you should do `conda create --name ta_python_${{ matrix.python_version }} python=${{ matrix.python_version }} -y